scripts/create_tables.sql

```diff
@@ -1 +1,29 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT
+);
+
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance BOOLEAN
+);
+
+CREATE TABLE appointments(
+  id SERIAL PRIMARY KEY,
+  'date' DATE,
```

Comment:
This line and the next will cause syntax errors.  You can use `date` and `time` without wrapping them with single quotation marks since we're only using them as column names rather than identifier names.